### ISSUE TITLE
Focus given to non-interactive div

### ISSUE THREAD

**Issue by greenyouse** [leandrowd/react-responsive-carousel#545]
Hi, I noticed a few a11y issues with this project so I wanted to make a few issues to help out. I can try to follow up with some PRs today too. Thank you for all the work you've done building and maintaining this! It's a really nice project to use :smile:

**Problem**:
`.carousel-root` has a `tabindex=0` attribute
Focus should not be given to non-interactive elements in the DOM.

**Impact**: 
Screen reader and keyboard only users will stop on this element but not be able to do anything with it since it's non-interactive. 
Also nothing will be read to screen readers when they land on this element. That will likely be confusing since each element that is focusable needs to serve a purpose for screen reader users.
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex#accessibility_concerns

**Recommended Solution**:
Remove the `tabindex=0` on this element since it serves no purpose for keyboard users or screen readers.

**Comment by leandrowd** [comment]
This will introduce other issues. I don't have the solution for you but keyboard navigation currently is only enabled if the carousel has focus. Removing that will make it not focusable. I'm happy to accept any accessibility improvements but please consider the impact...
